apps/portals/src/configurations/elportal/synapseConfigs/cohortbuilder.ts

```diff
@@ -80,6 +80,9 @@ export const filesView: SynapseConfig = {
     rgbIndex,
     name: 'Data',
     showExportToCavatica: true,
+    fileIdColumnName: 'fileId',
+    fileNameColumnName: 'fileId',
```

Comment:
Is ID as name expected or should we use the name (if that's currently available in the virtual table--maybe it needs to be updated)?

Reply:
Rather use the name once it's available in the Virtual Table (VT).  Currently this functionality is broken because the entity version has not been selected in the VT!